benchmark/simple_reporter.cc

```diff
@@ -0,0 +1,107 @@
+#include "benchmark/simple_reporter.h"
+
+#include <utility>
+
+#if defined(TACHYON_HAS_MATPLOTLIB)
+#include "third_party/matplotlibcpp17/include/pyplot.h"
+
+using namespace matplotlibcpp17;
+#endif  // defined(TACHYON_HAS_MATPLOTLIB)
+
+#include "tachyon/base/console/table_writer.h"
+#include "tachyon/base/containers/container_util.h"
+#include "tachyon/base/strings/string_number_conversions.h"
+
+namespace tachyon::benchmark {
+void SimpleReporter::AddTime(const std::string& vendor, double time) {
+  measurements_[vendor].push_back(time);
+}
+
+void SimpleReporter::AddTime(size_t vendor_idx, double time) {
+  AddTime(vendors_[vendor_idx], time);
+}
+
+void SimpleReporter::AddVendor(std::string_view vendor) {
+  vendors_.push_back(std::string(vendor));
+}
+
+void SimpleReporter::SetXLabel(std::string_view x_label) {
+  x_label_ = std::string(x_label);
+}
+
+void SimpleReporter::SetYLabel(std::string_view x_label) {
+  y_label_ = std::string(x_label);
+}
+
+void SimpleReporter::SetColumnLabels(std::vector<std::string>&& column_labels) {
+  column_labels_ = std::move(column_labels);
+}
```

Comment:
Why does this function enforce the move for its input?

Reply:
I will add the copy assign function.